[2024-03-17T01:02:50.996+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Airflow_Lab2_Flask.start_Flask_API manual__2024-03-17T01:02:49.899827+00:00 [queued]>
[2024-03-17T01:02:51.004+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Airflow_Lab2_Flask.start_Flask_API manual__2024-03-17T01:02:49.899827+00:00 [queued]>
[2024-03-17T01:02:51.004+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-03-17T01:02:51.004+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-03-17T01:02:51.004+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-03-17T01:02:51.014+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): start_Flask_API> on 2024-03-17 01:02:49.899827+00:00
[2024-03-17T01:02:51.017+0000] {standard_task_runner.py:55} INFO - Started process 315 to run task
[2024-03-17T01:02:51.020+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab2_Flask', 'start_Flask_API', 'manual__2024-03-17T01:02:49.899827+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/Flask_API.py', '--cfg-path', '/tmp/tmpvitu3klk']
[2024-03-17T01:02:51.020+0000] {standard_task_runner.py:83} INFO - Job 41: Subtask start_Flask_API
[2024-03-17T01:02:51.059+0000] {task_command.py:388} INFO - Running <TaskInstance: Airflow_Lab2_Flask.start_Flask_API manual__2024-03-17T01:02:49.899827+00:00 [running]> on host 3dffa19c86ca
[2024-03-17T01:02:51.132+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Airflow_Lab2_Flask
AIRFLOW_CTX_TASK_ID=start_Flask_API
AIRFLOW_CTX_EXECUTION_DATE=2024-03-17T01:02:49.899827+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-17T01:02:49.899827+00:00
[2024-03-17T01:02:51.138+0000] {logging_mixin.py:137} INFO -  * Serving Flask app 'unusual_prefix_fb1fa9258fa1ad786b4829a3672e98fa2e7542a0_Flask_API'
[2024-03-17T01:02:51.138+0000] {logging_mixin.py:137} INFO -  * Debug mode: off
[2024-03-17T01:02:51.139+0000] {logging_mixin.py:137} WARNING - Address already in use
[2024-03-17T01:02:51.139+0000] {logging_mixin.py:137} WARNING - Port 5001 is in use by another program. Either identify and stop that program, or start the server with a different port.
[2024-03-17T01:02:51.158+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-03-17T01:02:51.170+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
